aimsprop/plot.py

```diff
@@ -234,6 +234,10 @@ def plot_population(
 
     # Spaghetti plots from bundles
     Imap = {I: Iind for Iind, I in enumerate(bundle.Is)}
+    for bundle2 in bundles:
+        for Iind, I in enumerate(bundle2.Is):
+            if I not in Imap.keys():
+                Imap[I] = Iind
```

Comment:
@hayleyweir can you confirm this will maintain the intended plotting behavior already in the package?

Reply:
Looks good to me - it's not changing the plotting behaviour, it's just making sure that the dictionary is correct for the state colourings.